source/common/http/codec_client.h

```diff
@@ -120,6 +120,8 @@ class CodecClient : Logger::Loggable<Logger::Id::client>,
 
   const StreamInfo::StreamInfo& streamInfo() { return connection_->streamInfo(); }
 
+  Event::Dispatcher& dispatcher_;
```

Comment:
IIRC you can get the dispatcher directly from the connection and don't need this.